### DISCUSSION TITLE
Speed up creating arrowfiles

### DISCUSSION THREAD

**Discussion by Y-ogativity** [GreenleafLab/ArchR#1382] [General]
Hi, 

Thank you for developing the great package. I am analyzing a non-model organism whose genome is assembled into scaffolds and contigs. I have succeeded to import custom BS genome file and transcript annotation to the ArchR, but ~1.5 M contigs/scaffolds make it VERY slow to create arrow files;  ReadingTabixFile only 1 Percent takes overnight!! I thus would like to know how to accerralate reading fragments file so that I can make arrowfiles in a reasonable time. I hope that the attached ArchR log file will help you.

Regards,
Yohei Ogawa

[ArchR-createArrows-471115e63-Date-2022-04-10_Time-16-00-56.log](https://github.com/GreenleafLab/ArchR/files/8464565/ArchR-createArrows-471115e63-Date-2022-04-10_Time-16-00-56.log)
[session_Info.txt](https://github.com/GreenleafLab/ArchR/files/8464567/session_Info.txt)

**Comment by rcorces** [comment]
Hmmm. This is quite far outside of my expertise. I dont believe that there is anything on the ArchR side that can be done to make this go faster. I have a feeling that you might run into a lot of other downstream problems with ArchR with this type of genome as well since we have never had this use case before.

**Reply by Y-ogativity** [reply]
Thank you for the reply. 
OK, your comments make me think that I have to assemble the genome somehow and reduce the number of contigs/scaffolds....

**Reply by rcorces** [reply]
if possible, that sounds like the right approach.
